template-helpers/breach-detail.js

```diff
@@ -214,8 +203,14 @@ function getBreachDetail(args) {
       headline: LocaleUtils.fluentFormat(locales, "wtd-after-data-agg"),
       tips: getTips(locales, "data-agg"),
     };
-
-  } else {
+  }
+  if (breachDetail.categoryId === "sensitive-breach") {
```

Comment:
I'm not 100% certain on this `if...if...else` logic here.
But if I'm reading the code correctly here, we may have modified the current "data-aggregator-breach" logic.

If we come in on L197, and we're a "data-aggregator-breach", we do the `if`, and set `.whatIsThisBreach` and `.whatToDoTips` properties. Next on L207 we do a check for "sensitive-breach" and conditionally set some stuff, otherwise the `else` below on L213 will potentially overwrite our "data-aggregator-breach" `.whatIsThisBreach` config from L198-201 above.

Possible solutions are `if...else if...else`, or something like ye olde `switch`-a-roo:

```js
switch (breachDetail.categoryId) {
  case "data-aggregator-breach":
    // "what-is-data-agg" headline and copy.
    // modify `breachDetail.whatToDoTips`.
    break;
  case "sensitive-breach"
    // "sensitive-sites" headline and copy.
    break
  default:
    // "what-is-a-website-breach" and "website-breach-blurb" copy.
    break;
}
```